### DISCUSSION TITLE
Update to new version

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/Fooocus#1838] [General]
I installed Fooocus 3 weeks ago and I wonder what is the upgrade version process. For example, is bing Fooocus updated continuously? If so, how can upgrade fooocus without a full new instalación? How can I know the version fo Fooocus I have installed and he changes added to each new version? I guess there is also a policy about major versions? I guess many people knows how is done but I find difficult to figure out.

**Comment by poor7** [comment]
The version is listed here
`Fooocus/fooocus_version.py`

**Comment by xhoxye** [comment]
You can view this file Fooocus\update_log.md
![QQ截图20240110095456](https://github.com/lllyasviel/Fooocus/assets/129571231/1f79e74a-1858-4e8f-8474-255983fe168b)
![QQ截图20240110095546](https://github.com/lllyasviel/Fooocus/assets/129571231/1c3ef1e0-25f1-4b72-89b4-56b8fcd90b09)
![QQ截图20240110095734](https://github.com/lllyasviel/Fooocus/assets/129571231/905c8c2d-a24d-4543-b0d2-c2861197ba68)

**Comment by minus5m** [comment]
Thank you! Do you have to update it manually or does it update automatically though?

**Reply by mashb1t** [reply]
The update happens automatically when using `entry_with_update.py` as entrypoint, either by calling it directly or using run.bat etc.
Automatic updates may fail if you have local code changes, so please make sure to resolve merge conflicts if you encounter issues.
